inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/questions/RequestAnswer.java

```diff
@@ -0,0 +1,88 @@
+package uk.ac.ucl.rits.inform.informdb.questions;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import uk.ac.ucl.rits.inform.informdb.TemporalCore;
+import uk.ac.ucl.rits.inform.informdb.annotation.AuditTable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.time.Instant;
+
+/**
+ * Questions generated from specific data types (e.g. lab samples or consultation requests) trigger answers that or
+ * held as RequestAnswers. RequestAnswers therefore not only hold the answer as such but also link the answer to the
+ * question and the entity (i.e. data type) that triggered the question.
+ * @author Anika Cawthorn
+*/
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@AuditTable
+public class RequestAnswer extends TemporalCore<RequestAnswer, RequestAnswerAudit> {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long answerId;
+
+    @Column(columnDefinition = "text")
+    private String answer;
+
+    @ManyToOne
+    @JoinColumn(name = "questionId", nullable = false)
+    private Question questionId;
+
+    @Column(nullable = false)
+    private String parentTable;
+
+    @Column(nullable = false)
+    private long parentId;
+
+    public RequestAnswer() {}
+
+    /**
+     * Minimal request answer constructor that requires that the answer and the timestamps for when EMAP
+     * started processing this data type and from which time point the question information is valid from.
+     * @param questionId    Question this answer relates to
+     * @param answer        The actual question string linked to a data type
+     * @param parentTable   Data type that triggered question-answer pair creation, e.g. lab sample or consultation
+     *                      request
+     * @param parentId      Entity (e.g. lab sample or consultation request) that triggered the
```

Comment:
looks like we're missing the end here!
```suggestion
     * @param parentId      Entity (e.g. lab sample or consultation request) that triggered the
```